simplic-sql/Simplic.Sql/TypeHandler/PreciseDecimalHandler.cs

```diff
@@ -16,6 +17,10 @@ public override void SetValue(IDbDataParameter parameter, PreciseDecimal value)
 
         public override PreciseDecimal Parse(object value)
         {
+            if(value.GetType() == typeof(decimal))
```

Comment:
Hmm, `value is decimal` is much faster...

Reply:
https://stackoverflow.com/questions/686412/c-sharp-is-operator-performance

Reply:
@yvz5 the problem is, we have unboxing here and no cast...

Reply:
ok, i'm old school ...